tests/test_client.py

```diff
@@ -2572,3 +2572,8 @@ def test_status_check(client):
 def test_version_check(client):
     res = client.get("/_version")
     assert res.status_code == 200
+
+
+def test_get_dist(client):
+    res = client.get("/_dist")
+    assert res.status_code == 200
```

Comment:
let's `assert` the contents of the response are as expected, too (not that helpful for the current code, but will be if we reformat)